engine/src/exec-interface2.cpp

```diff
@@ -2480,8 +2480,18 @@ bool MCInterfaceStringCouldBeStack(MCStringRef p_string)
          MCStringBeginsWithCString(p_string, (const char_t *)"REVO",
                                   kMCCompareExact)))
         return true;
-    
-    // Check if it could be a script-only stack
+
+    //TODO: process BOM at the point of import into LiveCode (from URL)
+
+    //UTF-8 BOM would most likely result from a URL chunk.  It would
+    //not be evaluated any other way, so return true to have it evaluated
+    //as a script-only stack.
+    const char_t t_UTF8BOM[] = { 0xEF, 0xBB, 0xBF, 0x00 };
```

Comment:
My first attempt actually did that.  I backed it out since the parser did allow blank lines before script in another place.  I'll make these adjustments.